README.md

```diff
@@ -131,11 +131,12 @@ const sendTransaction = async (
   request: SinglePaymentRequest,
   provider: Program<HelioIdl>
 ): Promise<string | undefined> => {
+  const isHelioX = true
   try {
     if (symbol === "SOL") {
-      return await singleSolPaymentSC(provider, request);
+      return await singleSolPayment(provider, request, !isHelioX);
     }
-    return await singlePaymentSC(provider, request);
+    return await singlePayment(provider, request, !isHelioX);
```

Comment:
can you add a line here saying that:
`The transaction is also checked on the backend to verify if the user is a heliox member or not`